Source/Examples/WindowsForms/ExampleBrowser/MainForm.cs

```diff
@@ -51,10 +51,10 @@ private void FixTreeViewDpi()
         {
             using (var g = this.CreateGraphics())
             {
-                var scaleFactor = g.DpiY / 96f;
-                if (Math.Abs(scaleFactor - 1.25f) > 1e-5)
+                if (g.DpiY <= 96.0 && 144.0 <= g.DpiY)
```

Comment:
Are these numbers from observations/testing, or is there some sort of reference?

Reply:
Yes, this is from tests

Reply:
Could you add a comment to that effect? It's sure to confuse some more people in the future, and it's always a pain tracking down PR discussion.

Reply:
Good point ;)